airtime_mvc/composer.json

```diff
@@ -15,36 +15,36 @@
     "propel/propel1": "1.7.0-stable",
     "aws/aws-sdk-php": "2.7.9",
     "raven/raven": "0.12.0",
-    "massivescale/celery-php": "2.0.*@dev",
-    "simplepie/simplepie": "dev-master",
-    "zf1s/zend-application": "^1.12",
+    "massivescale/celery-php": "^2.1",
+    "simplepie/simplepie": "1.5.6",
+    "zf1s/zend-application": "^1.13",
     "composer/semver": "^1.4",
```

Comment:
I was able to bump semver to the latest version, as none of the breaking changes affect us.

Reply:
Bumped. If there are any others, we can probably include them in the PHP7.4 PR